### PR TITLE
Fork cmd/config to kpt and remove dependency

### PR DESCRIPTION
**Pull request by phanimarupaka**
This PR is to fork `cmd/config` cobra commands and remove dependency on it completely.

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -698,10 +698,12 @@ sigs.k8s.io/cli-utils v0.22.1-0.20201117031003-fd39030f0508 h1:uty/Bj8/fa2xX6CUp
 sigs.k8s.io/cli-utils v0.22.1-0.20201117031003-fd39030f0508/go.mod h1:Mt1gLc/Nfa7Z3Lhbfk72uT2Kc4GNyuX4oMqEN9FbPMs=
 sigs.k8s.io/controller-runtime v0.6.0 h1:Fzna3DY7c4BIP6KwfSlrfnj20DJ+SeMBK8HSFvOk9NM=
 sigs.k8s.io/controller-runtime v0.6.0/go.mod h1:CpYf5pdNY/B352A1TFLAS2JVSlnGQ5O2cftPHndTroo=
+sigs.k8s.io/kustomize v1.0.11 h1:Yb+6DDt9+aR2AvQApvUaKS/ugteeG4MPyoFeUHiPOjk=
 sigs.k8s.io/kustomize v2.0.3+incompatible h1:JUufWFNlI44MdtnjUqVnvh29rR37PQFzPbLXqhyOyX0=
 sigs.k8s.io/kustomize v2.0.3+incompatible/go.mod h1:MkjgH3RdOWrievjo6c9T245dYlB5QeXV4WCbnt/PEpU=
 sigs.k8s.io/kustomize/cmd/config v0.8.5 h1:e8uMA7LjAUss3cMefVoOY2tFr92brCTUAA6CmWgq57M=
 sigs.k8s.io/kustomize/cmd/config v0.8.5/go.mod h1:PlZTWxL7Xi75mY64HBC9IVf2xwrEbzy3L7r7+19kON4=
+sigs.k8s.io/kustomize/cmd/config v0.9.9 h1:UxYkJ60/VY7Oo1Ep19CrDmEL9wx72PgfnlKyPBsKof4=
```
Comment (mortent) [inline go.sum:706]: Why do we have this entry here?

File: go.mod [hunk 2]
```diff
@@ -22,6 +22,5 @@ require (
 	k8s.io/klog v1.0.0
 	k8s.io/kubectl v0.18.10
 	sigs.k8s.io/cli-utils v0.22.1-0.20201117031003-fd39030f0508
-	sigs.k8s.io/kustomize/cmd/config v0.8.5
```
Comment (droot) [inline go.mod:25]: Just ensure the version we forked is recorded in `thirdparty/...`

File: go.sum (hunk 1 above)
Comment (droot) [inline go.sum:706]: Surprised to see new references to `kustomize/cmd/config` here.

**Review by droot — APPROVED** [review]

File: go.sum (hunk 1 above)
Comment (phanimarupaka) [reply go.sum]: Cleaned it up.

Comment (phanimarupaka) [reply go.sum]: Cleaned it up.

File: go.mod (hunk 2 above)
Comment (phanimarupaka) [reply go.mod:25]: Done. Thanks.

**Pull request merged**